nemo/collections/common/parts/preprocessing/manifest.py

```diff
@@ -278,12 +279,14 @@ def get_full_path(
             # assume audio_file path is relative to data_dir
             audio_file_path = os.path.join(data_dir, audio_file)
 
-            if is_datastore_path(audio_file_path):
+            if is_datastore_path(audio_file_path) and cached:
                 # If audio was originally on an object store, use locally-cached path
```

Comment:
what is the expected behavior if `is_datastore_path(audio_file_path) == True` and `cached==False`?